pulp_rpm/tests/functional/constants.py

```diff
@@ -127,6 +127,14 @@
 .. NOTE:: This repository is not generated by pulp-fixtures.
 """
 
+FEDORA_MIRRORLIST_BASE = "https://mirrors.fedoraproject.org/mirrorlist"
+FEDORA_MIRRORLIST_PARAMS = "?repo=epel-modular-8&arch=x86_64&infra=stock&content=centos"
+RPM_EPEL_MIRROR_URL = FEDORA_MIRRORLIST_BASE + FEDORA_MIRRORLIST_PARAMS
```

Comment:
:+1:

Reply:
`FEDORA_MIRRORLIST_PARAMS` is maybe a misleading name?

Reply:
> `FEDORA_MIRRORLIST_PARAMS` is maybe a misleading or under-specified name?

It is? They're the parameters you can send along with a request for the mirrorlist from the fedoraproject

Reply:
They're specific to EPEL is what I mean